client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -334,6 +343,51 @@ class ContentDataProvider
     return success;
   }
 
+  public async handleCreationResponse(
+    resource: ContentItem,
+    newUri: Uri | undefined,
+    errorMessage: string,
+  ): Promise<void> {
+    if (!newUri) {
+      window.showErrorMessage(errorMessage);
+      return;
+    }
+
+    this.reveal(resource);
+  }
+
+  public async convertSasnbToFlw(
```

Comment:
Just duplicating this here so it gets the little resolve button :) 

> I've noticed some variables/errors/etc throughout the code that use flw as part of a variable name (i.e. InvalidFlwFileNameError). Can we update those references to flow (like InvalidFlowFileNameError).

In general, if function/variable/constant names can use the full names of things (i.e. SASNotebook, Flow), that would be ideal

Reply:
done in 9a191686d48ab21e54aa2247f2b7abd764b0d242